tests.py

```diff
@@ -188,3 +188,15 @@ def test_sip_api_already_set():
             import sip
             sip.setapi("QString", 1)
             assert_raises(ImportError, __import__, "Qt")
+
+
+def test_qtwebkit_pyside():
```

Comment:
Usually is better tests separated from each other (one test , one assert) , but of course there are exceptions.

Ops, noticed now, half of the test went away before the commit.